configuration/analyzer_config.json

```diff
@@ -410,6 +410,42 @@
     "soft_time_limit": 180,
     "queue": "long"
   },
+"InQuest_IOCdb": {
+  "type": "observable",
+  "observable_supported": ["ip", "url", "domain", "hash"],
+  "external_service": true,
+  "description": "Indicators of Compromise Database - search IOCdb",
+  "requires_configuration": true,
+  "python_module": "inquest.InQuest",
+  "additional_config_params": {
+    "api_key_name": "INQUEST_API_KEY",
+    "inquest_analysis" : "iocdb_search"
+  }
+},
+"InQuest_REPdb": {
+  "type": "observable",
+  "observable_supported": ["ip", "url", "domain", "hash"],
+  "external_service": true,
+  "description": "Reputation Database - search REPdb",
+  "requires_configuration": true,
+  "python_module": "inquest.InQuest",
+  "additional_config_params": {
+    "api_key_name": "INQUEST_API_KEY",
+    "inquest_analysis" : "repdb_search"
+  }
+},
+"InQuest_DFI": {
+  "type": "observable",
+  "observable_supported": ["ip", "url", "domain", "hash"],
```

Comment:
I think we can also add support for `generic` kind here. Goes for all 3 diff. configurations.

Reply:
DFI supports many different kinds of IOCs too. [reference to docs](https://labs.inquest.net/docs/?shell#api-dfi-search-ioc-kind)

Reply:
I think u added support for `generic` but forgot to put it inside `observable_supported`